### PR TITLE
Feature/issue 5 refactor

### PR DESCRIPTION
**Pull request by BethelBoy**
Alright, let's get this code refactored!! For consistency in how you were calling the presenter methods, I took away the `MilkInventory.sharedInstance` stuff and turned it into a `let model = MilkInventory()` at the top of the presenters so they can call the methods the same way the VC's are calling the presenter methods.

I also grouped the files together... but doesn't look like that's something that gets committed, which is interesting. Seems like the folder groups are a per user thing or something?? Not sure.

### CODE REVIEW

File: Pump-Tracker/MilkInventory.swift [hunk 1]
```diff
@@ -75,25 +70,25 @@ class MilkInventory {
...
                 // unschedule notification - currently we don't have notifications set up, but this will be used when that comes in
-                //                    if bag.notificationEnabled == true {
-                //                        UNUserNotificationCenter.current().removePendingNotificationRequests(withIdentifiers: [bag.number])
+                //                    if sku.notificationEnabled == true {
+                //                        UNUserNotificationCenter.current().removePendingNotificationRequests(withIdentifiers: [sku.number])
                 //                    }
-                // delete bag
-                context.delete(bag)
+                // delete sku
+                context.delete(sku)
```
Comment (beetdemguise) [inline Pump-Tracker/MilkInventory.swift]: Can add a `break` here for slight performance improvement.

**Comment by BethelBoy** [comment]
Agreed, good catch dude. Instead of needing to grab all inventory in those cases, I had actually implemented a way to pass the sku number into the `getAllInventory()` method when I initially set up core data, not sure why I didn't use that. So that's what I switched it to, no longer need to grab all inventory to do any looping for editing a sku. Woot. And also found there's an NSBatchDeleteRequest so that can handle deleting all data as well.

Unless I missed something, the `MilkInventory` class was the only place I found those loops.

**Comment by BethelBoy** [comment]
Test

File: Pump-Tracker/MilkInventory.swift [hunk 2]
```diff
@@ -96,35 +96,51 @@ class MilkInventory {
...
+            /// Create predicate to select record based on value of number property
+            request.predicate = NSPredicate(format: "id == %@", NSNumber(value: withID))
+            /// Get sku in the Inventory entity where id matches withID
+            sku = try context.fetch(request)
+        } catch {
+            /// TODO: Create real error handling
+            print("Error fetching data from CoreData")
+        }
+        
+        return sku[0]
```
Comment (beetdemguise) [inline Pump-Tracker/MilkInventory.swift]: This should probably go inside the `do`; that or we can check `sku.count` and return `nil` if we didn't find a sku with that ID.

File: Pump-Tracker/MilkInventory.swift [hunk 3]
```diff
@@ -14,131 +14,133 @@ import CoreData
...
+            request.predicate = NSPredicate(format: "id == %@", NSNumber(value: withID))
+            /// Get sku in the Inventory entity where id matches withID
+            sku = try context.fetch(request)
         } catch {
+            /// TODO: Create real error handling
             print("Error fetching data from CoreData")
         }
         
-        return bags
+        return sku[0]
```
Comment (beetdemguise) [inline Pump-Tracker/MilkInventory.swift]: This should probably go inside the do; that or we can check sku.count and return nil if we didn't find a sku with that ID.

**Pull request merged**